src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportUnitTest.java

```diff
@@ -41,17 +41,40 @@
                 {"Sample1\tfile1\t"},           // extra tab
                 {"Sample1\nfile"},              // newline instead of tab
                 {"\t"},                         // only tab
-                {"Sample1 file1"},              // non-tab whitespace
-                {"Sample 1\tfile1"},            // white space in a token
+                {"Sample1 file1"},              // 1 column
+                {" name name\tfile1"},          // preceding whitespace
+                {"name name \tfile1"},          // trailing whitespace
         };
     }
 
-    @Test(dataProvider = "getBadTestFiles", expectedExceptions = UserException.BadInput.class)
+    @Test(dataProvider = "getBadSampleNameMapFiles", expectedExceptions = UserException.BadInput.class)
     public void testBadInputFiles(final String text){
         final File sampleFile = IOUtils.writeTempFile(text, "badSampleMapping", ".txt");
         GenomicsDBImport.loadSampleNameMapFile(sampleFile.toPath()  );
     }
 
+    @DataProvider
+    public Object[][] getGoodSampleNameMapFileSyntax(){
+        return new Object[][]{
+                // Note: none of these files are real, these are just valid files syntactically
+                {"Sample1\tsamplePath \n"
+                +"Sample2\tsamplePath", new String[][] {{"Sample1","samplePath"},{"Sample1","samplePath"}}},     // normal sample names
```

Comment:
This expected value is wrong -- you check Sample1 twice, but don't check Sample2. Also, you should give Sample1 and Sample2 difference paths in the fake map.